pkg/volume/volume_linux.go

```diff
@@ -66,7 +66,7 @@ func SetVolumeOwnership(mounter Mounter, fsGroup *int64, fsGroupChangePolicy *v1
 	}
 
 	if skipPermissionChange(mounter, fsGroup, fsGroupChangePolicy) {
-		klog.V(3).Infof("skipping permission and ownership change for volume %s", mounter.GetPath())
+		klog.V(3).InfoS("Skipping permission and ownership change for volume", "volume", mounter.GetPath())
```

Comment:
```suggestion
		klog.V(3).InfoS("Skipping permission and ownership change for volume", "path", mounter.GetPath())
```
This log doesn't write volume name or volume object, but path where volume is located.